src/TLuaInterpreter.cpp

```diff
@@ -3307,6 +3307,44 @@ int TLuaInterpreter::createLabel(lua_State* L)
     return 1;
 }
 
+// Documentation: https://wiki.mudlet.org/w/Manual:Lua_Functions#createScrollBox
+int TLuaInterpreter::createScrollBox(lua_State* L)
+{
+    QString name = "";
+    int counter = 3;
+    //make the windowname optional by using counter. If windowname "main" add to main console
+
+    QString windowName = getVerifiedString(L, __func__, 1, "scrollBox name");
+    if (isMain(windowName)) {
+        // createScrollBox only accepts the empty name as the main window
```

Comment:
Why's that?

Reply:
The functions was copy/pasted from createMiniConsole and that one seems to work that way :wink: